Library/Formula/weechat.rb

```diff
@@ -28,7 +29,12 @@ class Weechat < Formula
   depends_on "curl" => :optional
 
   def install
-    args = std_cmake_args
```

Comment:
yes, I just have hard time using git. Need some help squashing stuff... trying to do it right now...